docs/src/app/components/pages/components/SelectField/ExampleCustomLabel.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import SelectField from 'material-ui/lib/SelectField';
+import MenuItem from 'material-ui/lib/menus/menu-item';
+
+export default class SelectFieldCustomLabelExample extends React.Component {
```

Comment:
I get what you mean. I'm only saying none of the other pages follow what you say. We should change them all at once. I don't like to see some follow and some not. Besides it still wouldn't match the file name `ExampleCustomLabel`. I think we should have a discussion on this when we want to move these files to their corresponding component folder later on. we are still new to this way of organizing the docs.